components/case/panelReference/index.tsx

```
@@ -1,82 +1,123 @@
+            })) || []
+          );
+        });
+
+        break;
+      }
+      case 'event:documentStatusUpdate': {
+        const { data: dataMsg } = message || {};
+
+        const { status, documentId } = dataMsg || {};
+
+        if (!!documentId) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)